docs/getting-started.md

```diff
@@ -21,6 +21,26 @@ The Client SDK is a set of user-facing programs to interact with Ledger and Audi
 ScalarDL (Ledger and Auditor) abstracts data as a set of assets, where each asset is composed of the history of a record identified by a key called `asset_id` and a historical version number called `age`.
 In this document, you will create a very simple application to manage an asset's status using ScalarDL Client SDK.
 
+## Install a JDK
+
+Because ScalarDL is written in Java, using Java is one of the easist ways to build your ScalarDL applications.
```

Comment:
@brfrn169 @josh-wong 
I followed the ScalarDB description, but the current one in ScalarDB is a bit misleading since it sounds like we must use Java to use ScalarDB.

Reply:
I think I understand. 

Basically, a JDK is required only because we're suggesting it as the easiest way to build applications even though there are other languages that we can use to build applications. Is that correct?

If so, I have just a couple minor changes to this sentence:

```suggestion
Because ScalarDL is written in Java, using Java is one of the easiest ways to build a ScalarDL application.
```

Reply:
> I followed the ScalarDB description, but the current one in ScalarDB is a bit misleading since it sounds like we must use Java to use ScalarDB.

Correct, our current documentation specifies that users must use Java:
https://github.com/scalar-labs/scalardb/blob/v3.11.0/docs/getting-started-with-scalardb.md#install-a-jdk

I believe that users are required to use Java when utilizing ScalarDB through the Java library (the community edition). However, Java is not required for using ScalarDB through the ScalarDB Cluster (the enterprise edition). Therefore, if users do not have a license for the enterprise edition, they must use Java.

@feeblefakie @josh-wong 
Does that make sense?

In that case, how about mentioning the ScalarDB Cluster thing (users can use non-Java languages through ScalarDB Cluster) as a note in the document?